### PR TITLE
HZIOS-1376 fix missing impression data in rewarded did fail to show

### PR DESCRIPTION
**Pull request by aseggemu-fyber**

### CODE REVIEW

File: Fyber FairBid/AdsScreenViewController.swift [hunk 1]
```diff
@@ -235,8 +235,8 @@ extension AdsScreenViewController: FYBRewardedDelegate {
     func rewardedDidShow(_ placementName: String, impressionData: FYBImpressionData) {
         addEventToCallbacksList(#function)
     }
-
-    func rewardedDidFail(toShow placementName: String, withError error: Error) {
+
```
Comment (saschabienert) [inline Fyber FairBid/AdsScreenViewController.swift:238]: you added some white spaces here

File: Fyber FairBid/AdsScreenViewController.swift [hunk 2]
```diff
@@ -235,8 +235,8 @@ extension AdsScreenViewController: FYBRewardedDelegate {
     func rewardedDidShow(_ placementName: String, impressionData: FYBImpressionData) {
         addEventToCallbacksList(#function)
     }
-
-    func rewardedDidFail(toShow placementName: String, withError error: Error) {
+    
+    func rewardedDidFail(toShow placementId: String, withError error: Error, impressionData: FYBImpressionData) {
```
Comment (saschabienert) [inline Fyber FairBid/AdsScreenViewController.swift:239]: I would make the `placementName` -> `placementId` change everywhere

**Review by saschabienert — CHANGES_REQUESTED** [review]

File: Fyber FairBid/AdsScreenViewController.swift (hunk 1 above)
Comment (aseggemu-fyber) [reply Fyber FairBid/AdsScreenViewController.swift]: sorry i did a tab and Xcode did an indentation.
I removed it in this commit 
1a2d4e4

File: Fyber FairBid/AdsScreenViewController.swift (hunk 2 above)
Comment (aseggemu-fyber) [reply Fyber FairBid/AdsScreenViewController.swift:239]: I refactored them in this commit 
c385997

**Pull request closed without merging**